apps/meteor/server/startup/migrations/v283.ts

```diff
@@ -12,5 +12,11 @@ addMigration({
 			// ignore
 			console.log(error);
 		}
+		try {
+			await LivechatPriority.col.dropIndex('name_1');
+		} catch (error) {
+			// ignore
+			console.log(error);
+		}
```

Comment:
Don't think we need this, since the change is not in develop (it won't even run on our dev machines since migration 283 should have already run)

Reply:
name has to stop being a unique field, or a lack of name will break the index

Reply:
Yeah, What I'm saying is that we don't need this index removal since the index is not created outside Alek, Yours and my machine 👀 

The only reason would be if QA envs have the index created, like if they reuse the DB, but not sure of it. Can we confirm that? 👀